projects/aca-content/folder-rules/src/manage-rules/manage-rules.smart-component.spec.ts

```diff
@@ -143,9 +168,9 @@ describe('ManageRulesSmartComponent', () => {
 
     expect(component).toBeTruthy();
 
-    const adfEmptyContent = debugElement.query(By.css('adf-empty-content'));
-    const ruleSets = debugElement.queryAll(By.css(`[data-automation-id="rule-set-list-item"]`));
-    const ruleDetails = debugElement.query(By.css('aca-rule-details'));
+    const adfEmptyContent = unitTestingUtils.getByCSS('adf-empty-content');
+    const ruleSets = unitTestingUtils.getAllByCSS(`[data-automation-id="rule-set-list-item"]`);
```

Comment:
here are couple other places where we use the same selector, please create some wrapping function or move selector to const and reuse